src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageGenerator.java

```diff
@@ -0,0 +1,80 @@
+package com.wikia.webdriver.common.core.imageutilities;
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
+import com.wikia.webdriver.common.core.CommonUtils;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.security.SecureRandom;
+
+import javax.imageio.ImageIO;
+
+/**
+ * @ownership: Quality Team
+ */
+public class ImageGenerator {
+
+  private String imagePath;
+  private String imageFolder = PageContent.IMAGE_UPLOAD_RESOURCES_PATH;
+  BufferedImage imageBuffer;
+  private int imageWidth;
+  private int imageHeight;
+
+  public ImageGenerator() {
+    imageWidth = 200;
```

Comment:
why not declare those values there https://github.com/Wikia/selenium-tests/pull/846/files#diff-bf8374f56664979c48c1a542fe88bf39R25?